tests/virtualization/universal/patch_guests.pm

```diff
@@ -22,11 +22,10 @@ sub reboot_guest {
     my $guest = shift;
     record_info("Rebooting $guest");
     if (get_var("KVM") || get_var("XEN")) {
-        script_run("rm /tmp/guests_ip/$guest");
         script_run("virsh shutdown $guest");
         script_retry("virsh domstate $guest|grep 'shut off'", retry => 5);
         script_run("virsh start $guest");
-        script_retry("test -f /tmp/guests_ip/$guest", retry => 5, delay => 60);
+        script_retry("nmap $guest -PN -p ssh | grep open", retry => 5, delay => 60);
```

Comment:
Hi @tbaev ,
I have a question here. It seems with above line of script_retry,  we no longer rely on `/tmp/guests_ip/$guest` to claim guest status. Then why do we still need the code piece in `initial_boot_notify.sh`?

Reply:
In our tests we use 2 ways to find if a guest is booted. One is if `ssh` is opened and the other one is with the `guests_ip` files.

 `guests_ip` test was crated because when installing a VM you can SSH to the VM before it is fully installed/booted and it will record the VM IP address. With current changes `guests_ip` will be created and checked only once, after initial installation of the VMs, after that for simple VM reboots we are using SSH to check if the VM is booted.